mitodl/ocw-studio
# enable tables in CKEditor

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #445
part of #569

#### What's this PR do?

This adds the CKEditor table plugin to our CKEditor configuration, giving us a table button and whatnot.

The markdown serialization is supported OOB because CKEditor's markdown stuff which we're already using already supports it.

#### How should this be manually tested?

Test out adding a table to a page. You should be able to do things like add text or numbers inside, maybe add some lines, and so on.

#### Screenshots (if appropriate)

<img width="786" alt="Screen Shot 2021-10-05 at 4 17 16 PM" src="https://user-images.githubusercontent.com/6207644/136096354-d1f4298a-dd0a-4203-81f6-6dcf2c7f2240.png">
<img width="811" alt="Screen Shot 2021-10-05 at 4 17 21 PM" src="https://user-images.githubusercontent.com/6207644/136096357-6c7ca246-8531-4ec7-8b87-d0d8927e3f43.png">
